packages/integrations/stores/ini-file/src/ini-file.ts

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
+import * as ini from 'ini';
+import { type Config } from '@configu/sdk';
+import { FileConfigStore } from '@configu/integrations/src/utils/File';
+
+export type IniFileConfigStoreConfiguration = { path: string };
+
+export class IniFileConfigStore extends FileConfigStore {
+    constructor({ path }: IniFileConfigStoreConfiguration) {
+        const initialFileState = '';
+        super('ini-file', { path, initialFileState });
```

Comment:
The implementation of `FileConfigStore` has changed, there's no need to pass in the `type` parameter any more. This can be fixed with:
```suggestion
        super({ path, initialFileState });
```